chapters/equations.tex

```diff
@@ -181,11 +181,20 @@ \subsection{If-Equations}\label{if-equations}
 In an equation section, the equations in the body are seen as equations that must be satisfied.
 The bodies that are not selected have no effect on that model evaluation.
 
-The \lstinline!if!-equations in equation sections which do not have exclusively parameter expressions as switching conditions shall have the same number of equations in each branch (a missing else is counted as zero equations and the number of equations is defined as in \cref{local-equation-size}).
+The \lstinline!if!-equations in equation sections which do not have exclusively parameter expressions as switching conditions shall satisfy the following:
```

Comment:
Let's take the opportunity to get rid of some redundancy, and avoid a potentially confusing reference (equation sections don't have switching conditions):

```suggestion
The \lstinline!if!-equations which do not have exclusively parameter expressions as switching conditions shall satisfy the following:
```

Reply:
Also, in https://github.com/modelica/ModelicaSpecification/issues/3779#issuecomment-3612632733 we said that this should be tied to evaluability, so I would have expected this:
```suggestion
The \lstinline!if!-equations which do not have exclusively evaluable expressions as switching conditions shall satisfy the following:
```